build/yamls/antrea-windows.yml

```diff
@@ -95,7 +95,7 @@ data:
     #                    supporting Pod traffic across IP subnets.
     # hybrid:            noEncap if source and destination Nodes are on the same subnet, otherwise encap.
     #
-    #trafficEncapMode: encap
```

Comment:
You will revert the change before merge?

Reply:
Yes, this change is to trigger CI test only.